docs/authentication/social-connections/custom.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Custom
+description: Configure a custom OpenID Connect (OIDC) compatible authentication provider for your Clerk application.
+---
+
+# Custom Auth Provider
+
+<Callout type="warning">
+  Custom auth providers are in beta. Please contact Support to get your application whitelisted for this feature.
+</Callout>
+
+Learn how to set up a custom authentication provider with Clerk.
+
+## Overview
+
+First, make sure your identity provider is compatible with OpenID Connect (OIDC), as Clerk relies on this protocol to authenticate users and retrieve their profile information.
+
+## Prerequisites
+
+- A Clerk Application created in your [Clerk Dashboard](https://dashboard.clerk.com/). For setup instructions, see our [Set up your application guide](/docs/quickstarts/setup-clerk).
+- Access to an OIDC-compliant identity provider.
+
+## Configuration Steps
+
+1. In the Clerk Dashboard, navigate to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)**.
+1. Under **Custom Auth Providers**, click **Add provider**.
+1. Fill in the following fields in the modal:
+
+- **Name**: The provider name (visible to users)
+- **Key**: A unique identifier for the provider (cannot be changed after creation)
+- **Discovery Endpoint**: The OIDC discovery endpoint URL of your provider
+- Alternatively, click **Manual configuration** to set up provider endpoints manually
```

Comment:
indent so that the bullet point is shown to be related to "Discovery Endpoint"

```suggestion
  - Alternatively, select **Manual configuration** to set up provider endpoints manually
```